db/migrate/20220204211051_add_details_to_comments.rb

```diff
@@ -0,0 +1,7 @@
+class AddDetailsToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_column :comments, :text, :string
```

Comment:
Please change the data type of the `text` attribute to `text` as it will be holding a few lines for the comment body but the `string` data type is for attributes such as `username`, and `password` etc.